SqlQueryBuilder/Select/IQueryBuilderOrderBy.cs

```diff
@@ -6,5 +6,6 @@ namespace SqlQueryBuilder.Select
     public interface IQueryBuilderOrderBy: IBuildQuery
     {
         IQueryBuilderOrderBy OrderBy<T>(Expression<Func<T, object>> lambda, bool desc = false, string tableNameAs = null);
+        IQueryBuilderOrderBy SkipTake(int skip, int take);
```

Comment:
Wouldn't it be better if Skip and Take were 2 different individual signatures? You might want to only use skip and then select all after those.